eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2FullFetchWithDeltaView.java

```diff
@@ -139,19 +140,32 @@ public RegistryFetch apply(List<ChangeNotification<InstanceInfo>> updates) {
 
             for (ChangeNotification<InstanceInfo> update : updates) {
                 com.netflix.appinfo.InstanceInfo v1Update = toEureka1xInstanceInfo(update.getData());
+                if (v1Update== null) {
```

Comment:
see change to toEureka1xInstanceInfo() which catches exceptions during conversion and returns null.